doc/source/whatsnew/v0.16.1.txt

```diff
@@ -27,7 +27,7 @@ Enhancements
 API changes
 ~~~~~~~~~~~
 
-
+- When subclassing ``DataFrame``, the correct class is returned, as specified in ``_constructor_sliced`` (PR 9632)
```

Comment:
Test lgtm. I think the issue is classified as a bug rather than API change. Can you describe a occurrence condition briefly, and refer to the issue as the same style as others (pls correct my English...)?

```
- Bug in subclassed ``DataFrame`` with ``MultiIndex`` columns may not return correct class (:issue:`9632`)
```